.github/PULL_REQUEST_TEMPLATE/new_post.md

```diff
@@ -0,0 +1,13 @@
+This PR adds a new post:
+
+# Content
+
+<!-- Short abstract/description of the post -->
+
+
+# Checklist
+
+ 1. [ ] No external resources (Check `README.md` how to add assets)
```

Comment:
I don't like this. We could argue that custom screenshots are not uploaded and linked to e.g. imgur, instead put them into the assets folder. But a general restriction could be related to a loss in post quality, since it refrains from using external well-cited content.

Reply:
Okay, resources might be the wrong wording, as I originally wanted to express that no external assets shall be used.

I did not mean resources in the sense of references (see 2nd point in checklist).

Reply:
Me aswell. Take an example:

>I want to write a post 
>There's a really good illustrations on the web
 >The illustration license does not allow distribution
>however, I am allowed to cite it

Because of the restriction I cannot use this illustration for my post. I need to drop it out or create an equivalent by myself.

Reply:
What about the following
```suggestion
 1. [ ] Every created and owned resource is located as an asset next to the post (Check `README.md` how to add assets). External resources are kept to a minimum
 2. [ ] Every external resource is properly cited
```

Reply:
The change is fine for me, but I am really unsure whether including an asset under a license where redistribution is not allowed is legal at all, eben when just including the original image.

Reply:
Should be covered by [§ 51 Urhg](https://dejure.org/gesetze/UrhG/51.html) and Fair Use

Reply:
Not a lawyer, but in that cases it's fine for me. I'll still prefer to do my on graphics, just to be sure, but in that case it's just a personal decision.